### PR TITLE
Bring app to release

### PR DESCRIPTION
**Pull request by julianuziemblo**
- hide missing features
- deal with all TODOs

related: https://github.com/orgs/RadioAktywne/projects/2/views/1?pane=issue&itemId=68197348&issue=RadioAktywne%7Cra-app%7C58

### CODE REVIEW

File: mobile/android/app/build.gradle [hunk 1]
```diff
@@ -40,21 +46,35 @@ android {
     }
 
     defaultConfig {
-        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.radioaktywne"
-        // You can update the following values to match your application needs.
-        // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
+        applicationId "pl.radioaktywne"
```
Comment (tymwitko) [inline mobile/android/app/build.gradle:49]: Did you consult this with @lewandowski-jan? A package name change could be crucial when it comes to publishing the app

Comment (tymwitko) [inline mobile/android/app/build.gradle:49]: Did you consult this with @lewandowski-jan? A package name change could be crucial when it comes to publishing the app

File: mobile/lib/components/newest_recording/newest_recording_widget.dart [hunk 2]
```diff
@@ -18,27 +18,30 @@ class NewestRecordingWidget extends StatelessWidget {
     final defaultShadowColor = context.colors.highlightGreen;
 
     return SwipeableCard(
-      // TODO: Change this for array of real SwipeableCardItems
```
Comment (tymwitko) [inline mobile/lib/components/newest_recording/newest_recording_widget.dart:21]: Don't remove that TODO, as the issue is not resolved - the `SwipeableCardItems` are still not "real"

File: mobile/lib/components/ra_playbutton.dart [hunk 3]
```diff
@@ -102,8 +102,6 @@ class _RaPlayButtonImage extends StatelessWidget {
           AudioProcessingState.loading ||
           AudioProcessingState.buffering =>
             _RaPlayButtonImageLoading(size: size),
-          // TODO: (maybe?) Handle error state:
-          // TODO: AudioProcessingState.error => ...,
```
Comment (tymwitko) [inline mobile/lib/components/ra_playbutton.dart:106]: Did this issue get resolved? I see no code changes related to that

File: mobile/lib/components/utility/lazy_loaded_grid_view.dart [hunk 4]
```diff
@@ -98,7 +98,6 @@ class LazyLoadedGridView<T> extends HookWidget {
               thumbnailPath: gridItem.thumbnailPath,
               titleOverlay: Text(
                 htmlUnescape.convert(gridItem.title),
-                // TODO: possibly context.textStyles.textSmallGreen, up to RA to decide
```
Comment (tymwitko) [inline mobile/lib/components/utility/lazy_loaded_grid_view.dart:101]: Did we get any confirmation?

File: mobile/lib/main.dart [hunk 5]
```diff
@@ -18,7 +18,6 @@ Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await SystemChrome.setPreferredOrientations([
     DeviceOrientation.portraitUp,
-    // DeviceOrientation.portraitDown, // TODO: decide about this one...
```
Comment (tymwitko) [inline mobile/lib/main.dart:21]: Again, any confirmation of that?

File: mobile/lib/models/ramowka_info.dart [hunk 6]
```diff
@@ -29,7 +29,7 @@ class RamowkaInfo implements Comparable<RamowkaInfo> {
   /// Parses title string to the required format
   static String parseTitle(String title) => htmlUnescape.convert(
         title
-            .replaceFirst('(Replay)', ' - powtórka') // TODO: use l10n
```
Comment (tymwitko) [inline mobile/lib/models/ramowka_info.dart:32]: L10n should still be used if possible (is it possible?)

File: mobile/lib/pages/recordings_page.dart [hunk 7]
```diff
@@ -36,25 +36,57 @@ class RecordingsPage extends HookWidget {
...
-          jsonData['source_url'] as String,
-          Duration(seconds: jsonData['media_details']['length'] as int)
-        ),
-      );
 
+    final recordingDetails = {
+      for (final RecordingInfo recording in recordings)
+        recording.title: RecordingDetails(),
+    };
+    // using title as key isn't ideal to put it mildly, but out of all the
```
Comment (tymwitko) [inline mobile/lib/pages/recordings_page.dart:44]: I suppose that asks for an `id` field to be added

File: mobile/lib/components/ra_burger_menu.dart [hunk 8]
```diff
@@ -31,10 +31,10 @@ class RaBurgerMenu extends HookWidget {
     (RaRoutes.recordings, 'Nagrania'),
     (RaRoutes.albumOfTheWeek, 'Płyta tygodnia'),
     (RaRoutes.articles, 'Publicystyka'),
-    (RaRoutes.radioPeople, 'Radiowcy'),
+    // (RaRoutes.radioPeople, 'Radiowcy'),
```
Comment (tymwitko) [inline mobile/lib/components/ra_burger_menu.dart:34]: You're commenting these out, yet in `ra_router_config` you've removed these components' navigation routes entirely - I suppose it'd be better to pick a lane

File: mobile/android/app/build.gradle (hunk 1 above)
Comment (julianuziemblo) [reply mobile/android/app/build.gradle:49]: Look which commit this is from. Its his change :D

Comment (tymwitko) [reply mobile/android/app/build.gradle:49]: Whoops, I see lol

File: mobile/lib/pages/recordings_page.dart (hunk 7 above)
Comment (julianuziemblo) [reply mobile/lib/pages/recordings_page.dart:44]: this is a change that should affect the whole project not just the release, so I fixed it here: https://github.com/RadioAktywne/ra-app/pull/98

File: mobile/android/prod_key.properties
Comment (Kszesiek) [inline mobile/android/prod_key.properties:1]: Is this a good idea to keep this file on a public repo? Shouldn't it be excluded in .gitignore or sth? Asking purely out of intuition

**Review by Kszesiek — COMMENTED** [review]
All seems fine, apart from one detail. Didn't manage to run it though (weird flutter gradle issue, don't have time to handle this right now unfortunately). Read through all the code changes though.

File: mobile/android/prod_key.properties
Comment (julianuziemblo) [reply mobile/android/prod_key.properties:1]: It's not a good practice, but according to @lewandowski-jan, we can keep it for now and change it with prod version. 

I've added an issue for this (#99)

**Review by Kszesiek — APPROVED** [review]
Let's goooooo

**Pull request merged**